repos/corefx.proj

```diff
@@ -3,11 +3,11 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))/dir.props" />
   <PropertyGroup>
     <BuildArguments>-$(Configuration) -buildArch=$(Platform) -portable=$(PortableBuild) -BuildTests=false</BuildArguments>
-    <BuildCommand>$(ProjectDirectory)/build$(ShellExtension) $(BuildArguments) -- /p:ILLinkTrimAssembly=false</BuildCommand>
+    <BuildCommand>$(ProjectDirectory)/build$(ShellExtension) $(BuildArguments) -- /p:ILLinkTrimAssembly=false /p:DotNetAssetRootUrl=file:%2F%2F$(SourceBuiltAssetsDir)</BuildCommand>
```

Comment:
What's `/p:DotNetAssetRootUrl` for in corefx?

Reply:
I was attempting to fix a new warning, I'll back this out for now.